canonical-websites/www.ubuntu.com
# 18.10

## Done

- All the changes for 18.10

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/) or the [demo](http://www.ubuntu.com-canonical-websites-pr-4235.run.demo.haus/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Check the:
- [x] /download pages
- [x] /desktop/developer pages
- [x] /server page
- [x] the homepage takeover

## Issue / Card

Fixes #4234
